content/docs/user-guide/managing-external-data.md

```diff
@@ -1,13 +1,14 @@
 # External Outputs
 
 > ⚠️ This is an advanced feature for very specific situations and not
-> recommended except if there's absolutely no other alternative. In most cases
-> alternatives like the
-> [to-cache](/doc/command-reference/add#example-transfer-to-the-cache) or
-> [to-remote](/doc/command-reference/add#example-transfer-to-remote-storage)
-> strategies of `dvc add` and `dvc import-url` are more convenient. **Note**
-> that external outputs are not pushed or pulled from/to
+> recommended except if there's absolutely no other alternative. Note that
+> external outputs are not pushed or pulled from/to
 > [remote storage](/doc/command-reference/remote).
+>
+> In most cases the
+> [to-cache](/doc/command-reference/add#example-transfer-to-an-external-cache)
+> or [to-remote](/doc/command-reference/add#example-transfer-to-remote-storage)
+> strategies of `dvc add` and `dvc import-url` are more convenient.
```

Comment:
don't like `are more convenient.` but I guess we'll address it later. It's not about convenience to my mind. It's about different workflows around data. And we need to explain to people that in certain cases (unless you indeed write/read directly from/to remote location from code) you don't need this. If you are looking for a way to add data to DVC in regular way (remote storage, dvc pull, dvc push , whatever - find a way to define it ) there are better options

Reply:
Agreed. Changing to "better" for now.